hack/generate-deletion-protected-seed-crds.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
```

Comment:
Instead of copying https://github.com/gardener/gardener/blob/master/hack/generate-seed-crds.sh, can you rather make it reusable for your case?